pilz_extensions/test/unittest_joint_limit.cpp

```diff
@@ -43,6 +46,38 @@ TEST_F(JointLimitTest, SimpleRead) {
   EXPECT_EQ(-1, joint_limits_extended.max_deceleration);
 }
 
+/**
+ * @brief Test reading a non-existing joint limit
```

Comment:
I think this can be figured from "readNonExistingJointLimit"

Reply:
Done